timesketch/lib/analyzers/contrib/misp_analyzer_test.py

```diff
@@ -43,7 +49,7 @@ def setUp(self):
     @mock.patch("requests.post")
     def test_attr_match(self, mock_requests_post):
         """Test match"""
-        analyzer = misp_analyzer.MispAnalyzer("test_index", 1)
+        analyzer = misp_analyzer.MispAnalyzer("test_index", 1, None, **QUERY_MISP)
         analyzer.misp_url = "https://test.com/"
         analyzer.misp_api_key = "test"
```

Comment:
You have defined the data already in the set-up function above. No need for those two lines.